WcaOnRails/app/views/posts/_specialbox.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="ui card">
+  <div class="content">
+    <h3 class="center aligned header"><%= title %></h3>
```

Comment:
Can you change it so that the `New` text does not push the actual title out of the center?
```
|_____CENTERED NEW_____|
```
should be more like
```
|_______CENTERED NEW___|
```